processor/src/chiplets/mod.rs

```diff
@@ -285,16 +285,16 @@ impl Chiplets {
                     let rest = bitwise_fragment.push_column_slice(rest, bitwise.trace_len());
                     memory_fragment.push_column_slice(rest, memory.trace_len());
                 },
-                4 | 11..=14 => {
+                4 | 10..=14 => {
                     // columns 4 - 11 to 14 are relevant for hasher, bitwise, memory chiplets and
```

Comment:
```suggestion
                    // columns 4 - 10 to 14 are relevant for hasher, bitwise, memory chiplets and
```

To be honest these comments are not very useful and they are bound to be de-synchronized with the code.

Reply:
Thanks for catching that. I agree that these a very brittle. My hope is that once we start treating the trace as a row-major matrix, we can cast rows of a trace to a more descriptive `struct` allowing for direct reference of the column values.